### ISSUE TITLE
Порог Урона и Урон

### ISSUE THREAD

**Issue by Enclave2020** [animusdev/Fallout-13#32]
Урон проходит через броню с шансом. То есть, ты стреляешь по силовой броне, или даже лучше - бьешь её ножем, и с шансом (кстати, довольно неплохим), ты можешь её ПРОБИТЬ. И нанести ПОЛНЫЙ урон. А половина урона наносится чуть ли не большую часть времени

Вместо ребалансировки я предлагаю более адекватное решение - взять уже готовую систему из F2 \ Нью Вегаса, вырезав уменьшение урона после порога
http://fallout.wikia.com/wiki/DT

Откуда брать инфу по каждой броне?
Легко - на той же вики. Там есть информация по КАЖДОЙ вещи что есть в игре, вот кримеру АПА МК2 (параметр DT справа)
http://fallout.wikia.com/wiki/Advanced_power_armor_(Fallout_2)

То же касается и оружия. Чертова плазменная винтовка, невероятный хайтек в мире фоллыча, дамажит НЕМНОГО больше обычного оружия. Почему? Потому что под капотом лазер с ТГ.
Урон штурмовой винтовки, по канону 18 (http://fallout.wikia.com/wiki/Service_rifle)
У плазмача же 47 (http://fallout.wikia.com/wiki/Plasma_rifle_(Fallout%3A_New_Vegas))
Ощутимая разница, не так ли?

**Comment by ak72ti** [comment]
Насчет материалов, это ты хорошо придумал, но тут следует учитывать такой нюанс, что фоллыч все таки РПГ, и пунктов хп у персонажа с силой в 10, например, намного более 100.
Потому, я в основном расчитываю урон по схеме сравнения ХП фоллыча нашего, с оригиналом. Частенько потому урон делится на два. Ближе к завершению работ по оружию, перестрелки будут очень близки по духу к фоллычу, как если бы вы сражались с противником одного уровня с вами. Сейчас это в процессе, но сильно заторможено, поскольку я хочу оптимизировать систему выдачи профессии и починить наконец лендмарки, если этого за меня конечно, коната не сделает.

**Comment by ak72ti** [comment]
Напомню, что урон той же монтировки по канону или шесть или 8, что меньше даже урона плитки пола (10).

**Comment by ak72ti** [comment]
Следовательно, тут придется совершать ковровую нарезку урона с учетом этой формулы, а не только менять броню и делать неебический дамаг.

**Comment by Enclave2020** [comment]
@ak72ti Точно! Я забыл, что в F2 изначальное HP около 25, соответственно, все оружие меньше урона, как и защита брони. Но сравнивал броню из F2 с оружием NV
Выходит у силовой брони ПУ 28 (http://fallout.wikia.com/wiki/Remnants_power_armor)

В Нью Вегасе, насколько я помню, защита еще более примитивная. У брони есть ЕДИНЫЙ ПУ (защита от лазеров, ударов и прочего одинаковая)
Все данные на википедии являются базовой, то есть, по сути, из неё уже исключены уровни, которых у нас и нет

**Вот несколько расчетов:**
Рейдер нападает с плазменной винтовкой (47) на Анклав (28), соответственно, урон 19, убьет он его через 6 выстрелов
Легионер нападает с мачете (11) на паладина БС в Т51 (25), он её не пробивает и погибает
Житель города с 10ММ пистолетом (22) нападает на человека в боевой броне (15), урон 7, убивает его за 15 выстрелов

Причем, Мачете и 10ММ это довольно слабое оружие
Например, снайперская винтовка наносит 45 урона

**Как временное решение**, могу предложить переход на систему F3, или что-то похожее. Суть в том, что там броня **гарантированно** поглащала % урона. По сути, сейчас можно просто заменить эту систему со случайностью (поглощения \ половины урона \ пробивания) на постоянное снижение урона по процентам
